test/unit-test/shadow_utest.c

```diff
@@ -445,6 +455,16 @@
  */
 #define TEST_NAMED_TOPIC_LENGTH_INVALID_GET_REJECTED         ( ( uint16_t ) sizeof( TEST_NAMED_TOPIC_STRING_INVALID_GET_REJECTED ) - 1U )
 
+/**
+ * @brief The length of #TEST_NAMED_TOPIC_STRING_EXCEEDS_MAX_THING_NAME.
+ */
+#define TEST_NAMED_TOPIC_LENGTH_EXCEEDS_MAX_THING_NAME       ( ( uint16_t ) sizeof( TEST_NAMED_TOPIC_STRING_EXCEEDS_MAX_THING_NAME ) - 1U )
+
+/**
+ * @brief The length of #TEST_NAMED_TOPIC_STRING_EXCEEDS_MAX_SHADOW_NAME.
+ */
+#define TEST_NAMED_TOPIC_LENGTH_EXCEEDS_MAX_SHADOW_NAME      ( ( uint16_t ) sizeof( TEST_NAMED_TOPIC_STRING_EXCEEDS_MAX_SHADOW_NAME ) - 1U )
+
```

Comment:
Do we have test cases for empty thing/shadow names?  e.g.:

```c
define TEST_NAMED_TOPIC_STRING_EMPTY_THING     "$aws/things//shadow/name/TestShadowName/delete/rejected"
define TEST_NAMED_TOPIC_STRING_EMPTY_SHADOW    "$aws/things/ThingName/shadow/name//delete/rejected"
```

Reply:
Yes. We have test cases for empty shadowName and Thingname
Empty Thing name topic - TEST_TOPIC_STRING_INVALID_THINGNAME
Empty Shadow name topic - TEST_NAMED_TOPIC_STRING_INVALID_SHADOWNAME